sql/execution_pipeline_impl.h

```diff
@@ -125,6 +125,103 @@ struct GenerateRowsElement: public PipelineElement {
     std::shared_ptr<BoundPipelineElement> bind() const;
 };
 
+/*****************************************************************************/
+/* SUB SELECT LEXICAL SCOPE                                                  */
+/*****************************************************************************/
+
+/** Lexical scope for a sub select.  It allows for the output of the SELECT to be
+    used in wildcards (SELECT * from (SELECT 1 AS X))
+*/
+
+struct SubSelectLexicalScope: public TableLexicalScope {
```

Comment:
Should add to the comment what its output looks like, ie what elements it will add when it runs.